plugins/k8s/resoto_plugin_k8s/deferred_edges/__init__.py

```diff
@@ -0,0 +1,33 @@
+from resotolib.baseresources import BaseResource
+from resotolib.graph import Graph
+from resotolib.logger import log
+from resoto_plugin_k8s.deferred_edges.aws import (
+    link_k8s_node_to_aws_nodegroup_or_ec2_instance,
+    link_k8s_cluster_to_eks_cluster,
+    link_pv_to_ebs_volume,
+    link_service_to_elb,
+)
+from resoto_plugin_k8s.deferred_edges.digitalocean import (
+    link_node_to_do_droplet,
+    link_service_to_do_lb,
+    link_pv_to_do_volume,
+)
+
+
+def create_deferred_edges(graph: Graph) -> None:
+    if not isinstance(graph, Graph):
+        log.error(f"Expected type Graph, got {type(graph)}")
+        return
+    for node in graph.nodes:
+        if not isinstance(node, BaseResource):
+            log.warning(f"Node {node} is not a BaseResource")
+            continue
+        # AWS
```

Comment:
Is there a chance to detect if we are in EKS or DO? Would be great to only add deferred edges if required.

Reply:
Maybe, but I would make that an improvement in another PR. This PR just migrates the existing post_collect logic without changing it.

I could add a config.resotoworker detection to see if AWS is even activated and if not not run the AWS linking code. Though that seems error prone, because I could have one worker where AWS is running and another where K8S is running and would want them to link deferred edges properly.